frontend/packages/console-shared/src/components/spotlight/spotlight.scss

```diff
@@ -0,0 +1,64 @@
+@keyframes co-spotlight-expand {
+  0% {
+    outline-offset: -4px;
+    outline-width: 4px;
+    opacity: 1;
+  }
+  100% {
+    outline-offset: 21px;
+    outline-width: 12px;
+    opacity: 0;
+  }
+}
+
+@keyframes co-spotlight-fade-in {
+  0% {
+    opacity: 0;
+  }
+  100% {
+    opacity: 1;
+  }
+}
+
+@keyframes co-spotlight-fade-out {
+  0% {
+    opacity: 1;
+  }
+  100% {
+    opacity: 0;
+  }
+}
+
+.ocs-spotlight {
+  pointer-events: none;
+  position: absolute;
```

Comment:
I don't think this `position` is needed either since you're not specifying the `top`/`left` esk attributes.

Reply:
this is being used by guided tours as well which uses top/left.

Reply:
🤔  Pretty sure if you don't use `top` / `left` of 0 it does wonky things... but okay. Let's leave it and worry about it if it's an issue.